### ISSUE TITLE
Quick Question

### ISSUE THREAD

**Issue by matthew-d-stringer** [team401/2017-Vision-Code#12]
Hey, I have a quick question, you know that jar file inside your lib folder, is that the zeromq library for java?Thank you.

**Comment by camearle20** [comment]
Yes it is!  For the roborio side of vision we use JeroMQ, which is a 100% Java implementation of the ZeroMQ protocol.  We used ZMQ because of how simple it is to get things going, as well as it's versatile selection of supported languages and platforms.
